app/views/api/services/new.html.slim

```diff
@@ -1,6 +1,6 @@
 h2 Create New Service
 
-- if ThreeScale.config.service_discovery.enabled
+- if service_discovery_accessible?
```

Comment:
Hmm. I think we can't remove `ThreeScale.config.service_discovery.enabled`. Maybe include it in https://github.com/3scale/porta/blob/f25f15d5c58058ae8349d801754efb9d546453a2/app/lib/service_discovery/controller_methods.rb#L21.

Reply:
It is already inside of the TokenRetriever

First it checks if it is enabled then if it is accessible (server is reachable more or less)